spec/support/landing_page_behavior.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+shared_examples 'a landing page' do
```

Comment:
We can leave this, but these shared examples, associated context, and associated specs seem like a lot of unnecessary duplication and overhead for our spec suite. The `LandingPageFinder` specs should be testing all the potential landing pages for different types of users, so the only additional  tests we should need are:
- specs to verify that the `UserSessionsController` is performing a redirection (which we have)
- specs to verify that the `OmniauthCallbacksController` is performing a redirection (which we have via the Cucumber tests)
- enough cucumber tests to verify that we're redirecting different types of users to different landing pages (which we have)

Reply:
Leaving alone for now, in the interest of getting this ticket done.